packages/api-core/src/monetize/fetchers.ts

```diff
@@ -0,0 +1,10 @@
+import { wpcom } from '../wpcom-fetcher';
+import { MembershipSubscription } from './types';
+
+export async function fetchMonetizeSubscriptions(): Promise< MembershipSubscription[] > {
```

Comment:
The file should ideally be called `api-core/me-memberships.ts`. See: pgz0xU-df-p2

Reply:
I moved it to `api-core/src/me-monetize`